engine/repositories/processor_push.go

```diff
@@ -30,6 +30,11 @@ func (s *Service) processPush(ctx context.Context, op *sdk.Operation) error {
 		return sdk.WrapError(err, "unable to process gitclone")
 	}
 
+	// FIXME create Fetch and FetchTags method in go repo
```

Comment:
?

Reply:
For now I used FetchRemoteBranch but as I only want to fetch, It could be nice to update go repo to prevent usage of FetchRemoteBranch, I think that it's not so easy to understand what this func do.